runelite-client/src/main/java/net/runelite/client/plugins/devtools/BorderOverlay.java

```diff
@@ -0,0 +1,195 @@
+package net.runelite.client.plugins.devtools;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.geom.GeneralPath;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.api.Perspective;
+import net.runelite.api.Point;
+import net.runelite.api.coords.LocalPoint;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+
+public class BorderOverlay extends Overlay
+{
+	private static final Color GREEN = Color.GREEN;
+	private static final Color BLUE = Color.BLUE;
```

Comment:
This naming probably can be changed to be more descriptive for what is it. E.g GREEN could be CHUNK_BORDER_COLOR, and BLUE MAP_SQUARE_COLOR or something like that.